tools/osbuild-depsolve-dnf5

```diff
@@ -357,10 +358,12 @@ class Solver():
                 "repo_id": package.get_repo_id(),
                 "path": package.get_location(),
                 "remote_location": remote_location(package),
-                "checksum": (
-                    f"{package.get_checksum().get_type_str()}:"
-                    f"{package.get_checksum().get_checksum()}"
-                )
+                "checksum": f"{package.get_checksum().get_type_str()}:{package.get_checksum().get_checksum()}",
+                "sslverify": pkg_repo_cfg.get_sslverify_option().get_value(),
```

Comment:
dnf5 doesn't need .get_value() anymore, they fixed that last year, so you can just use the result of the function call.

Reply:
I did that at first but was getting errors on Fedora 39 about it not being serialisable.

For example:
```diff
diff --git a/tools/osbuild-depsolve-dnf5 b/tools/osbuild-depsolve-dnf5
index 16501bcd..6b366efe 100755
--- a/tools/osbuild-depsolve-dnf5
+++ b/tools/osbuild-depsolve-dnf5
@@ -359,7 +359,7 @@ class Solver():
                 "path": package.get_location(),
                 "remote_location": remote_location(package),
                 "checksum": f"{package.get_checksum().get_type_str()}:{package.get_checksum().get_checksum()}",
-                "sslverify": pkg_repo_cfg.get_sslverify_option().get_value(),
+                "sslverify": pkg_repo_cfg.get_sslverify_option(),
                 "gpgcheck": pkg_repo_cfg.get_gpgcheck_option().get_value(),
                 "sslclientkey": pkg_repo_cfg.get_sslclientkey_option().get_value(),
                 "sslclientcert": pkg_repo_cfg.get_sslclientcert_option().get_value(),
```
causes
```
TypeError: Object of type OptionChildBool is not JSON serializable
```
with libdnf5 version 5.1.14

Reply:
Odd, they must have missed some places when they changed it.